src/scss/darkmode_variables.scss

```diff
@@ -0,0 +1,89 @@
+
+/* EXPERIMENTAL */
+.dark-mode {
+    /* Background Color */
+    --oc-grey-900: #f8fafc;
+    --oc-grey-800: #f1f5f9;
+    --oc-grey-700: #e2e8f0;
+    --oc-grey-600: #cbd5e1;
+    --oc-grey-500: #94a3b8;
+    --oc-grey-400: #64748b;
+    --oc-grey-300: #475569;
+    --oc-grey-200: #334155;
+    --oc-grey-100: #1e293b;
+    --oc-grey-50: #0f172a;
+    --oc-grey-10: #020617;
+
+    --oc-contrast-black: #000000;
+    --oc-contrast-white: #ffffff;
+
+    --oc-text-000: #000000;
+    --oc-text-200: var(--oc-grey-300);
+    --oc-text-300: var(--oc-grey-500);
+    --oc-text-400: var(--oc-grey-600);
+    --oc-text-500: var(--oc-grey-900);
+
+    --oc-background-dark: #020617;
+    --oc-background-light: #000000;
```

Comment:
Also here doesn't `--oc-background-light` color is more darker then `--oc-background-dark`, it may cause some misunderstanding to other dev